pages/boac/cohort_list_view_page.rb

```diff
@@ -15,7 +15,7 @@ class CohortListViewPage < CohortPage
 
       elements(:player_link, :link, xpath: '//ul[@id="cohort-members-list"]/a')
       elements(:player_name, :h3, xpath: '//ul[@id="cohort-members-list"]//h3')
-      elements(:player_sid, :div, xpath: '//ul[@id="cohort-members-list"]//div[@data-ng-bind="student.sid"]')
+      elements(:player_sid, :div, xpath: '//ul[@id="cohort-members-list"]//div[contains(@class, "cohort-member-sid")]')
```

Comment:
@pfarestveit  I apologize in advance...  All `cohort-member-*` class names were changed to `student-*`.  So the xpath above should be `student-sid`